python/lsst/analysis/drp/scatterPlot.py

```diff
@@ -512,29 +527,55 @@ def scatterPlotWithTwoHists(self, catPlot, plotInfo, sumStats, yLims=False, xLim
                 highThresh = self.config.highSnStatisticSelectorActions.statSelector.threshold
                 statText = f"S/N > {highThresh} Stats ({magCol} < {highMags[sourceType]})\n"
                 statText += highStats[sourceType]
-                fig.text(xPos, 0.087, statText, bbox=bbox, transform=fig.transFigure, fontsize=6)
-                if highMags[sourceType] != "-":
```

Comment:
Could you remind/point to me when these would be set as `-` and why they are not needed anymore?

Reply:
This gets set to "-" when there aren't any sources to do stats on. This if statement isn't needed because changing to f strings deals with it.